README.md

```diff
@@ -17,9 +17,19 @@ add-on.
 
 ## Contributing changes
 
-**Pull Requests should use the `master` branch by default. Only make Pull Requests against other branches (e.g. `2.1` or `3.0`) if your changes only apply to that specific version of Godot.**
+If you'd like to contribute to the official documentation, whether you want to improve existing pages or write a new one, we would ask you to follow our workflow and guidelines:
 
-Though arguably less convenient to edit than a wiki, this Git repository is meant to receive pull requests to always improve the documentation, add new pages, etc. Having direct access to the source files in a revision control system is a big plus to ensure the quality of our documentation.
+1. If you want to write a new page, please start by opening an issue or join the discussion on an open issue with your proposal. This ensures that your work matches our [content guidelines](https://docs.godotengine.org/en/latest/community/contributing/content_guidelines.html) and saves both you and the reviewers' time.
```

Comment:
There are multiple reviewers, you know 🙂

Reply:
Agreed, it could refer to both one or many reviewers. I wouldn't say "yours and reviewers' time" is correct, but I would be happy with either of the other two.

Reply:
I'm not a native speaker, but I think what's intended and which should be grammatically correct here is:
i`and saves both you and the reviewers time`, i.e. it saves time for you and for the reviewers. (Not necessarily "your" and "theirs", but just indefinite "time" which is being saved for both parties.)

Reply:
Well, yeah, it can be that too. The only problem I currently see is that it's a mix of "it saves whom and whose time", which is incorrect either way. Should be one or the other.

Reply:
how about "it saves everyone time"? It's simpler that way